resources/game_config.json

```diff
@@ -250,7 +250,7 @@
 		"best_stat_modifier": 10,
 		"fail_stat_cat_modifier": -15,
 		"chance_of_romance_patrol": 16,
-		"debug_ensure_patrol_id": null,
+		"debug_ensure_patrol_id": "bch_bord_camptest",
```

Comment:
You've left the testing patrol active here!

Reply:
Ooop! my bad XD will fix